src/BenchmarkDotNet/Helpers/AsciiHelper.cs

```diff
@@ -0,0 +1,12 @@
+using JetBrains.Annotations;
+
+namespace BenchmarkDotNet.Helpers
+{
+    internal static class AsciiHelper
+    {
+        public static string ToAscii([CanBeNull] this string s)
+        {
+            return s?.Replace("\u03BC", "u");
```

Comment:
it would be nice to have some comment here explaining what is `\u03BC` and why we need to replace it

are there any other characters that we should take care of?

Reply:
> it would be nice to have some comment here explaining what is \u03BC and why we need to replace it

Good point, fixed.

> are there any other characters that we should take care of?

Currently, `\u03BC` is the only symbol that should be patched.